packages/vuetify/src/components/VDataTable/VDataTableHeaderDesktop.ts

```diff
@@ -16,76 +16,95 @@ export default mixins(header).extend({
         },
       }, ['group'])
     },
-    // eslint-disable-next-line max-statements
+    getAria (beingSorted: boolean, isDesc: boolean) {
+      let ariaSort = 'none'
+      let ariaLabel = [
+        this.$vuetify.lang.t('$vuetify.dataTable.ariaLabel.sortNone'),
+        this.$vuetify.lang.t('$vuetify.dataTable.ariaLabel.activateAscending'),
+      ]
+
+      if (beingSorted) {
+        if (isDesc) {
+          ariaSort = 'descending'
+          ariaLabel = [
```

Comment:
```ts
const $t = str => this.$vuetify.lang.t(`$vuetify.dataTable.${str}`) // Should probably create a function to avoid having to write so much.
const activated = this.options.mustSort ? 'activeAscending' : 'activateNone'

ariaLabel.push($t('sortDescending'))
ariaLabel.push($t(activated))
```